remoteit/desktop
# [RC-1591] log history paywall

### Description

 log history paywall

### Test plan



1. Sign in
2. Go to devices
3. Click in logs
4. You should log with the limit of your plan

### Ticket

https://remoteit.atlassian.net/browse/RC-1591
